### PR TITLE
Added German translation Strings.

### PR DESCRIPTION
**Pull request by peterge1998**
Updates some existing Strings and added some missing ones. Already included both Error messages (error_unreachable_server and error_login_credentials) for PR #40.

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -153,4 +154,32 @@
...
+    <string name="action_sort_method">Sortierungsmethode</string>
+    <string name="action_new_playlist">Neue Wiedergabeliste</string>
+    <string name="black_theme_name">Schwarz</string>
+    <string name="pref_header_images">Bilder</string>
+    <string name="pref_title_maximum_bitrate">Maximale Bitrate</string>
+    <string name="pref_title_remember_queue">Warteschlange merken</string>
+    <string name="pref_title_maximum_list_size">Maximale Listen Größe</string>
+    <string name="pref_title_images_cache_size">Cache Größe</string>
+    <string name="pref_title_images_external_directory">Externes Verzeichnis</string>
+    <string name="pref_summary_images_external_directory">"Bildcache in externen Verzeichnis speichern, um die Dateien zwischen Installationen aufzubewahren. "</string>
```
Comment (actuallynotjohndoe) [inline app/src/main/res/values-de/strings.xml:172]:
```suggestion
    <string name="pref_summary_images_external_directory">"Bildcache in externem Verzeichnis speichern, um die Dateien zwischen Installationen aufzubewahren. "</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -153,4 +154,32 @@
...
+    <string name="report_bugs">Melde Fehler</string>
+    <string name="report_bugs_summary">Einen Issue für neue Funktionen oder Fehler öffnen.</string>
+    <string name="logout">Abmelden</string>
+    <string name="unlimited">Unbegrenzt</string>
+    <string name="sort_method_random">Zufällig</string>
+    <string name="added_title_to_queue">Einen Titel zur Wiedergabeliste hinzugefügt.</string>
+    <string name="added_x_titles_to_queue">%1$d Titel zur Wiedergabeliste hinzugefügt.</string>
+    <string name="error_version">Bitte update den Server auf die aktuellste Version.</string>
+    <string name="error_share_file">Die Datei konnte nicht geteilt werden.</string>
+    <string name="pref_title_transcode_codec">Transkodierungs Codec</string>
```
Comment (actuallynotjohndoe) [inline app/src/main/res/values-de/strings.xml:184]:
```suggestion
    <string name="pref_title_transcode_codec">Transkodierungs-Codec</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -153,4 +154,32 @@
...
+    <string name="action_clear_queue">Warteschlange leeren</string>
+    <string name="action_save_queue">Warteschlange speichern</string>
+    <string name="action_sort_method">Sortierungsmethode</string>
+    <string name="action_new_playlist">Neue Wiedergabeliste</string>
+    <string name="black_theme_name">Schwarz</string>
+    <string name="pref_header_images">Bilder</string>
+    <string name="pref_title_maximum_bitrate">Maximale Bitrate</string>
+    <string name="pref_title_remember_queue">Warteschlange merken</string>
+    <string name="pref_title_maximum_list_size">Maximale Listen Größe</string>
+    <string name="pref_title_images_cache_size">Cache Größe</string>
```
Comment (actuallynotjohndoe) [inline app/src/main/res/values-de/strings.xml:170]:
```suggestion
    <string name="pref_title_images_cache_size">Cache-Größe</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -153,4 +154,32 @@
...
+    <string name="action_add_to_playlist">Zu Wiedergabeliste hinzufügen</string>
+    <string name="action_clear_queue">Warteschlange leeren</string>
+    <string name="action_save_queue">Warteschlange speichern</string>
+    <string name="action_sort_method">Sortierungsmethode</string>
+    <string name="action_new_playlist">Neue Wiedergabeliste</string>
+    <string name="black_theme_name">Schwarz</string>
+    <string name="pref_header_images">Bilder</string>
+    <string name="pref_title_maximum_bitrate">Maximale Bitrate</string>
+    <string name="pref_title_remember_queue">Warteschlange merken</string>
+    <string name="pref_title_maximum_list_size">Maximale Listen Größe</string>
```
Comment (actuallynotjohndoe) [inline app/src/main/res/values-de/strings.xml:169]:
```suggestion
    <string name="pref_title_maximum_list_size">Maximale Listengröße</string>
```

**Comment by dkanada** [comment]
Can you squash these commits into a single one?

**Pull request closed without merging**